packages/mobile/src/components/track-list/TablePlayButton.tsx

```diff
@@ -0,0 +1,55 @@
+import { StyleProp, TouchableOpacity, View, ViewStyle } from 'react-native'
+
+import IconPause from 'app/assets/images/pbIconPause.svg'
+import IconPlay from 'app/assets/images/pbIconPlay.svg'
+import { makeStyles } from 'app/styles'
+
+type TablePlayButtonProps = {
+  playing?: boolean
+  paused: boolean
+  hideDefault?: boolean
+  onPress?: () => void
+  styles?: {
+    root: StyleProp<ViewStyle>
+    svg: StyleProp<ViewStyle>
+  }
+}
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  root: {},
+  svg: {},
+  hideDefault: {}
+}))
+
+export const TablePlayButton = ({
+  playing = false,
+  paused,
+  hideDefault = true,
+  onPress = () => {},
+  styles: propStyles = {
```

Comment:
i like the default arg here, what we have been doing in other componets is just `stylesProp?.root`, what do you think?

Reply:
Yea I should have checked, sounds good to me